### PR TITLE
SE-2084 Updated Markdown for NB

### PR DESCRIPTION
**Pull request by DavidWassellFinbourne**

### CODE REVIEW

File: examples/use-cases/valuation/Valuations with Inferred FX Rates.ipynb [hunk 1]
```diff
@@ -33,7 +33,7 @@
      "name": "stderr",
```
Comment (steco) [inline examples/use-cases/valuation/Valuations with Inferred FX Rates.ipynb:33]: <p>typo, plus my comment from the other MR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (steco) [inline examples/use-cases/valuation/Valuations with Inferred FX Rates.ipynb:33]: <p>" values" -&gt; "prices" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (steco) [inline examples/use-cases/valuation/Valuations with Inferred FX Rates.ipynb:33]: <p>space between "LUSID" and "inverts"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (DavidWassellFinbourne) [reply examples/use-cases/valuation/Valuations with Inferred FX Rates.ipynb:33]: <p>No space in nb</p><p> </p>

**Pull request merged**